snowplow_tracker/emitters.py

```diff
@@ -386,16 +480,23 @@ def __init__(
         :type  thread_count: int
         :param byte_limit:  The size event list after reaching which queued events will be flushed
         :type  byte_limit:  int | None
+        :param max_retry_delay_seconds:     Set the maximum time between attempts to send failed events to the collector. Default 60 seconds
+        :type max_retry_delay_seconds:      int
+        :param buffer_capacity: The maximum capacity of the event buffer. The default buffer capacity is 10,000 events.
+                                When the buffer is full new events are lost.
+        :type buffer_capacity: int 
         """
         super(AsyncEmitter, self).__init__(
             endpoint,
             protocol,
             port,
             method,
-            buffer_size,
```

Comment:
I can't add a note below, so adding here: from the `sync_flush()` method of the `AsyncEmitter` we should call `self._flush_now()` instead `self.flush()` to make sure that the flush is synchronous and ignores the retry delay.